app/controllers/application_controller.rb

```diff
@@ -15,4 +15,8 @@ def current_user
   def current_shopper?
     !(current_user && (current_user.admin? || current_user.merchant?))
   end
+  
+  def current_admin?
```

Comment:
@plapicola why did we move this here from the base controller? just trying to understand!

Reply:
By putting it in the application controller and making it a helper method, we're able to use it in the views, including ones outside the admin namespace where it existed before. For this case specifically I did it so that I can control if all merchants(including disabled merchants) are displayed on the merchant index based on the current user's role. Hope this helps!